.github/workflows/start-mysql.sh

```diff
@@ -29,7 +29,41 @@ wait_for_mysql() {
   echo "MySQL at port $port has started!"
 }
 
+wait_for_version () {
```

Comment:
We need to check for two different states

1. can we connect and retrieve the correct version
2. can we validate that the health check succeeds.